src/cljc/rems/common/form.cljc

```diff
@@ -239,6 +246,25 @@
       (assoc form :form/fields fields)
       form)))
 
+;; yaya, this one fails
+(deftest validate-fields-compatibility-test
```

Comment:
Tests, especially tests like this that are in source files, are usually named like `test-validate-fields-compatiblity`. That way they don't interfere with e.g. function name completion.

Reply:
It is deleted now